### PR TITLE
Slider

### PR DESCRIPTION
**Pull request by JulienDelemarle**
create slider

### CODE REVIEW

File: src/components/Slider/Slider.jsx [hunk 1]
```diff
@@ -1,3 +1,84 @@
-export default function Slider() {
-  return <div>Slider</div>;
+import React from 'react';
+import { Slide } from 'react-slideshow-image';
+import 'react-slideshow-image/dist/styles.css';
+import PropTypes from 'prop-types';
```
Comment (GeorgesFORTRY) [inline src/components/Slider/Slider.jsx]: no need proptypes

File: src/components/Slider/Style.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+const Sslider = styled.div`
+  width: 100vw;
+  padding-top: 20px;
+  margin: auto;
+
+  .each-slide > div {
+    display: flex;
+    align-items: center;
+    justify-content: left;
+
```
Comment (GeorgesFORTRY) [inline src/components/Slider/Style.jsx]: empty line

File: src/components/Slider/Style.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    flex-direction: column;
+    align-items: center;
+    text-align: justify;
+    justify-content: center;
+    color: white;
+  }
+
+  .slider-text {
+    display: flex;
+
```
Comment (GeorgesFORTRY) [inline src/components/Slider/Style.jsx]: empty line

File: src/components/Slider/Style.jsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  }
+  /* .each-slideshow-indicator::before {
+    background: ${({ theme }) => theme.fontColor || 'yellowgreen'};
+  }
+
+  .style {
+    height: 75vh;
+    width: 100%;
+  } 
+  */
```
Comment (GeorgesFORTRY) [inline src/components/Slider/Style.jsx]: death code

File: src/components/Slider/Slider.jsx [hunk 5]
```diff
@@ -1,3 +1,84 @@
...
+        </div>
+      </Slide>
+    </Sslider>
+  );
 }
+
+Slider.propTypes = {
+  theme: PropTypes.string.isRequired,
+};
+
```
Comment (GeorgesFORTRY) [inline src/components/Slider/Slider.jsx]: you don't need proptypes without props

**Review by GeorgesFORTRY — CHANGES_REQUESTED** [review]
some bugs to fix

**Review by GeorgesFORTRY — DISMISSED** [review]
ok no more problems

File: src/components/Slider/Slider.jsx [hunk 6]
```diff
@@ -1,3 +1,79 @@
...
+              </h2>
+              <p className="slider-text">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+                eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+                enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                nisi ut aliquip ex ea commodo consequat.
+              </p>
+            </span>
+          </div>
+        </div>
```
Comment (LoicBrassart) [inline src/components/Slider/Slider.jsx]: This block could be 66% shorter by using a `slideImages.map()` (and would work with *any* number of slides)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
See content

**Review by GeorgesFORTRY — DISMISSED** [review]
better 👍

**Review by LoicBrassart — DISMISSED** [review]
NmE!

**Review by GeorgesFORTRY — APPROVED** [review]
ok no conflict

File: .gitignore [hunk 7]
```diff
@@ -22,3 +22,6 @@ package-lock.json
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+edTechHdf.vuerd
```
Comment (LoicBrassart) [inline .gitignore:26]: Why would your vuerd file be in the frontend repo ?

**Review by LoicBrassart — APPROVED** [review]
One minor question, but nevermind that's ok!

**Pull request merged**